### PR TITLE
new icon: netlify (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #1814: [ICON REQUEST]: Netlify
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

A remote-first cloud computing company that offers a development platform that includes build, deploy, and serverless backend services for web applications and dynamic websites.

### Links and sources

https://www.netlify.com

### Additional information

`#2,280` Global rank on Similar Web. https://www.similarweb.com/website/netlify.app

### PR DESCRIPTION
**Pull request by Gautam25Raj**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1814**

## Link to prove your SVG is correct and up-to-date.
https://www.netlify.com/blog/netlify-new-logo/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -6056,6 +6056,30 @@
...
         "aliases": []
     },
+    {
+        "name": "netlify",
+        "altnames": [],
+        "tags": [
+            "cloud hosting",
+            "serverless",
+            "dynamic websites",
+            "web applications"
```
Comment (Panquesito7) [inline devicon.json:6066]: They have a [GitHub organization](https://github.com/netlify) with a few open-source projects.
```suggestion
            "web applications",
            "open-source"
```

**Review by Gautam25Raj — COMMENTED** [review]
All done. Everything looks fine I guess.

**Review by Panquesito7 — APPROVED** [review]
Great work. Thanks! 🚀

image

image

image

image

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️ 
Thank you for the contribution! 🙇
Your icons look incredibly clean! Great work, especially on the plain version!

**Pull request merged**